test/e2e/far_e2e_test.go

```diff
@@ -197,39 +224,86 @@ func buildNodeParameters(clusterPlatformType configv1.PlatformType) (map[v1alpha
 			fmt.Printf("can't get nodes' information - AWS instance ID\n")
 			return nil, err
 		}
-		nodeIdentifier = v1alpha1.ParameterName("--plug")
+		nodeIdentifier = v1alpha1.ParameterName(nodeIdentifierPrefixAWS)
 
 	} else if clusterPlatformType == configv1.BareMetalPlatformType {
 		nodeListParam, err = e2eUtils.GetBMHNodeInfoList(machineClient)
 		if err != nil {
 			fmt.Printf("can't get nodes' information - ports\n")
 			return nil, err
 		}
-		nodeIdentifier = v1alpha1.ParameterName("--ipport")
+		nodeIdentifier = v1alpha1.ParameterName(nodeIdentifierPrefixIPMI)
 	}
 	testNodeParam = map[v1alpha1.ParameterName]map[v1alpha1.NodeName]string{nodeIdentifier: nodeListParam}
 	return testNodeParam, nil
 }
 
-// checkFarLogs gets the FAR pod and checks whether it's logs have logString
-func checkFarLogs(logString string) {
-	var pod *corev1.Pod
-	var err error
-	EventuallyWithOffset(1, func() *corev1.Pod {
-		pod, err = utils.GetFenceAgentsRemediationPod(k8sClient)
+// getNodeBootTime returns the bootime of node nodeName if possible, otherwise it returns an error
+func getNodeBootTime(nodeName string) (time.Time, error) {
+	bootTime, err := e2eUtils.GetBootTime(clientSet, nodeName, testNsName, log)
+	if bootTime != nil && err == nil {
+		return *bootTime, nil
+	}
+	return time.Time{}, err
+}
+
+// getFarPod gets a podList with all the pods that match by FAR's label
+func getFarPod() *corev1.PodList {
+	var (
+		pod *corev1.PodList
+		err error
+	)
+	EventuallyWithOffset(1, func() *corev1.PodList {
+		pod, err = utils.GetFARPodList(k8sClient)
```

Comment:
why do you use the list here, wouldn't GetFenceAgentsRemediationPod() be the better choice? Do we need this function at all? :confused:

Reply:
No, it is redundant with the new update to GetFenceAgentsRemediationPod() function